tools/AzureRM.BootStrapper/AzureRM.Bootstrapper.psm1

```diff
@@ -1055,3 +1084,112 @@ function Set-BootstrapRepo
 	param([string]$Repo)
 	$script:BootStrapRepo = $Repo
 }
+
+<#
+.ExternalHelp help\AzureRM.Bootstrapper-help.xml 
+#>
+function Set-AzureRmDefaultProfile
+{
+  [CmdletBinding(SupportsShouldProcess = $true)]
+  param()
+  DynamicParam
+  {
+    $params = New-Object -Type System.Management.Automation.RuntimeDefinedParameterDictionary
+    Add-ProfileParam $params
+    Add-ForceParam $params
+    Add-ScopeParam $params
+    return $params
+  }
+  PROCESS {
+    $armProfile = $PSBoundParameters.Profile
+    $Scope = $PSBoundParameters.Scope
+    $Force = $PSBoundParameters.Force
+    
+    $defaultProfile = $Global:PSDefaultParameterValues["*-AzureRmProfile:Profile"]
+    if ($defaultProfile -ne $armProfile)
+    {
+      # Set DefaultProfile for this session
+      $Global:PSDefaultParameterValues["*-AzureRmProfile:Profile"]="$armProfile"
+
+      # Edit the profile content
+      $profileContent = @"
+          `$PSDefaultParameterValues["*-AzureRmProfile:Profile"]="$armProfile"
```

Comment:
I am not following what the intention is here.  This just sets the default for AzureRmProfile cmdlets.  This does not set the default version of cmdlets that will be loaded when PowerShell is opened.

If we wanted to set the default cmdlet version loaded at runtime, we would need to change the default parameter for ```Import-Module```, probably with a script block, selecting the version based on the module name